pkg/operator/certrotation/target.go

```diff
@@ -31,6 +31,7 @@ type TargetRotation struct {
 
 	ClientRotation  *ClientRotation
 	ServingRotation *ServingRotation
+	SignerRotation  *SignerRotation
```

Comment:
godoc for all three

Reply:
You are building a sum-type here. One could put the actual signer call into a method of each and implement and interface here, hence only have one rotation object here.